src/containers/Home/Home.js

```diff
@@ -11,9 +13,15 @@ class Home extends Component {
             <AuxComponent>
                 <div className={classes.Home}>
                     <h1>Select Game Mode</h1>
-                    <GameMode gameMode="Classic" history={this.props.history} />
-                    <GameMode gameMode="Timed" history={this.props.history} />
-                    <GameMode gameMode="Survival" history={this.props.history} />
+                    <div className={classes.Btn_group}>
```

Comment:
Should we get rid of the New Game and Leaderboards nav items and instead add one more button here for Leaderboards?